files/12dSolutions/Georef_A_12d.ifc

```diff
@@ -0,0 +1,53 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[Ifc4x4NotAssigned]'), '2;1');
+FILE_NAME ('Georef_A_12d.ifc', '2023-03-16T14:45:22', ('Adam Mackellar'), ('12d Solutions Pty Ltd.'), 'IFC Engine rev 1389', '12d Model 15.0C1h [DEVELOP] INT Build 02-03-2023', 'Lee Gregory');
+FILE_SCHEMA (('IFC4X4_a0bae928'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('2SJXx4QHf3dwcBsduY2aig', #2, 'IfcTunnelDeployment', 'IfcTunnelDeployment workgroup', $, $, $, (#13), #8);
+#2 = IFCOWNERHISTORY(#3, #7, $, .NOCHANGE., $, $, $, 1678938322);
+#3 = IFCPERSONANDORGANIZATION(#4, #6, $);
+#4 = IFCPERSON($, 'Mackellar', 'Adam', $, ('Mr.'), $, (#5), $);
+#5 = IFCACTORROLE(.USERDEFINED., 'Software Engineer', $);
+#6 = IFCORGANIZATION('12d', '12d Solutions Pty Ltd.', $, $, $);
+#7 = IFCAPPLICATION(#6, '15.0C1h [DEVELOP] INT Build 02-03-2023', '12d Model', '12d Model');
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT('3D', 'Model', 3, 1.E-6, #15, #19);
+#14 = IFCLOCALPLACEMENT($, #15);
+#15 = IFCAXIS2PLACEMENT3D(#16, #17, #18);
+#16 = IFCCARTESIANPOINT((0., 0., 0.));
+#17 = IFCDIRECTION((0., 0., 1.));
+#18 = IFCDIRECTION((1., 0., 0.));
+#19 = IFCDIRECTION((0., 1.));
+#20 = IFCPROJECTEDCRS('EPSG:3857', $, 'World Geodetic System 1984 ensemble', $, $, $, #9);
+#21 = IFCMAPCONVERSIONSCALED(#13, #20, 16833194.0548, -4009644.0757, 0., 1., 0., 1., 1., 1., 1.);
```

Comment:
Is this a specific point on the map, or are number chosen at random?

Also, I would love to see an Australian EPSG used in practice here, and not something that is [21km precise on ground](https://epsg.io/3857) :)

Reply:
I think that should be the Sydney Opera House.